resources/js/components/Dashboard/UserDahboardInfrastructure.vue

```diff
@@ -5,26 +5,26 @@
         </div>
          <div class="card-body pt-3" style="height: 18em;overflow: auto;">
             <ul v-if="this.billingDetails.current_amount" class="list list-group unstyled-list">
-                <li class="d-flex justify-content-between" style="font-size:16px;">
-                    <span> Unbilled amount till today </span> 
+                <li style="font-size:16px;">
+                    <span class="d-block"> Unbilled amount till today </span> 
```

Comment:
Could you please try using this
```suggestion
                    <div> Unbilled amount till today </div> 
```